packages/prism/src/index.js

```diff
@@ -4,18 +4,17 @@
 import Highlight, { defaultProps } from 'prism-react-renderer'
 import { jsx, Styled } from 'theme-ui'
 
-export default ({ children, className, title }) => {
-  const [language] = className.replace(/language-/, '').split(' ')
+export default ({ children, className: outerClassName, title }) => {
+  const [language] = outerClassName.replace(/language-/, '').split(' ')
 
   return (
     <Highlight
       {...defaultProps}
       code={children.trim()}
       language={language}
-      theme={undefined}
-    >
+      theme={undefined}>
       {({ className, style, tokens, getLineProps, getTokenProps }) => (
-        <Styled.pre className={className} sx={style}>
+        <Styled.pre className={`${outerClassName} ${className}`} sx={style}>
```

Comment:
Also looks like this should be an inline style:
```suggestion
        <Styled.pre className={`${outerClassName} ${className}`} style={style}>
```